sources/engine/Stride.Engine/Updater/ArrayUpdateResolver.cs

```diff
@@ -6,7 +6,7 @@
 namespace Stride.Updater
 {
     /// <summary>
-    /// Resolver for <see cref="T[]"/> in property path.
+    /// Resolver for arrays of type <see cref="Array"/> with elements of type <typeparamref name="T"/>.
```

Comment:
Nitpick: `arrays of type <see cref="Array"/>` seems a bit redundant, arrays are always of that type unless I'm missing something ?

``Resolver for <see cref="Array"/> with elements of type <typeparamref name="T"/>.`` should be good enough.

Reply:
Any feedback welcome! Thanks, I will me correctins.